### DISCUSSION TITLE
Event on backup failure or success

### DISCUSSION THREAD

**Discussion by tieskuh** [lavinir/hassio-onedrive-backup#20] [Ideas]
Hi, thanks for the great add-on. 
I have a suggestion/idea: It would be great if an event was created on backup failure and success. Somewhat like this: https://jcwillox.github.io/hass-auto-backup/events/#example-automation-using-events

**Comment by lavinir** [comment]
Thanks for the suggestion. While no dedicated event is fired currently, you do have the option to toggle on [Persistent Notifications](https://github.com/lavinir/hassio-onedrive-backup#notify_on_error) in the addon in case of error. and it will show up under your HA notifications. If needed it is also possible to set up an automation based on those notifications (I have something like this set up in Node Red where any persistent notification is also sent to my phone via push notification). 

Would that serve what you need?

**Comment by tieskuh** [comment]
Yeah that works but an event would be more flexible. Maybe something for a future update.

**Reply by lavinir** [reply]
This is now available in version 1.3

**Reply by tieskuh** [reply]
Great! Thank you very much!
